src/main/java/org/prebid/server/bidder/adf/AdfBidder.java

```diff
@@ -49,17 +53,30 @@ public AdfBidder(String endpointUrl, JacksonMapper mapper) {
     public Result<List<HttpRequest<BidRequest>>> makeHttpRequests(BidRequest bidRequest) {
         final List<Imp> modifiedImps = new ArrayList<>();
         final List<BidderError> errors = new ArrayList<>();
+        String priceType = "";
 
         for (Imp imp : bidRequest.getImp()) {
             try {
                 final ExtImpAdf adfImp = parseExt(imp);
                 final Imp modifiedImp = imp.toBuilder().tagid(adfImp.getMid()).build();
                 modifiedImps.add(modifiedImp);
+                priceType = Objects.nonNull(adfImp.getPriceType()) ? adfImp.getPriceType() : "";
             } catch (PreBidException e) {
                 errors.add(BidderError.badInput(e.getMessage()));
             }
         }
 
+        if (StringUtils.isNotEmpty(priceType)) {
+            final ObjectNode adfNode = mapper.mapper().createObjectNode()
+                    .put("pt", priceType);
+            final ExtRequestPrebid extRequestPrebid =
+                    ObjectUtil.getIfNotNull(bidRequest.getExt(), ExtRequest::getPrebid);
+            final ExtRequest extRequest = mapper.fillExtension(
+                    ExtRequest.of(extRequestPrebid), adfNode);
```

Comment:
Following the current solution, you can wipe the data in `request.getExt()` outside of `request.getExt().getPrebid()`. Try like this:
```
final ExtRequest extRequest = ObjectUtils.defaultIfNull(bidRequest.getExt(), ExtRequest.empty());
mapper.fillExtension(extRequest, adfNode);
```

Reply:
Pls follow the above approach